src/python/foglamp/core/scheduler/scheduler.py

```diff
@@ -1169,9 +918,9 @@ def _resume_check_schedules(self):
             raise ValueError("name can not be empty")
 
         if schedule.repeat is not None and not isinstance(schedule.repeat, datetime.timedelta):
-            raise ValueError('repeat must be of type datetime.time')
+            raise ValueError('repeat must be of type datetime.timedelta')
 
-        if schedule.exclusive is None:
+        if schedule.exclusive is None or not (schedule.exclusive == True or schedule.exclusive == False):
```

Comment:
could you please elaborate why? you mean it can be non-boolean as well? then add type check ?